scripts/dream_web.py

```diff
@@ -56,44 +57,64 @@ def do_POST(self):
 
         print(f"Request to generate with prompt: {prompt}")
 
-        outputs = []
+        def image_done(image, seed):
+            config = post_data.copy() # Shallow copy
+            config['initimg'] = ''
+
+            # Write PNGs
+            pngwriter = PngWriter(
+                "./outputs/img-samples/", config['prompt'], 1
+            )
+            # metadata_str = f'prompt2png({json.dumps(config)} seed={seed}' # gets written into the PNG
```

Comment:
Is this being done elsewhere (not understanding why this is commented out)?

Reply:
This variable is defined here: https://github.com/lstein/stable-diffusion/blob/main/ldm/simplet2i.py#L178, but never used. I'm not sure if it was planned for the future, but just for completeness sake I added it here as well.

Reply:
This seems to be a hanging statement that should have been removed. Feel free to delete the comment. I will delete L178 from simplet2i.py.